src/util/time.h

```diff
@@ -6,24 +6,43 @@
 #ifndef BITCOIN_UTIL_TIME_H
 #define BITCOIN_UTIL_TIME_H
 
+#include <compat.h>
+
+#include <chrono>
 #include <stdint.h>
 #include <string>
-#include <chrono>
+
 #include <unordered_map>
+#include <sync.h>
 
-#include "sync.h"
+using namespace std::chrono_literals;
```

Comment:
I don't think you want `using namespace` in the header. That will expose the namespace for everyone including this which, in the end, is probably everyone.

Reply:
That is straight from upstream.... I think it is so in every file you can do stuff like 1s, 1m, etc., and yes you would want that globally.

Reply:
I can try ripping it out if you want, but that is the way Bitcoin has it.